### DISCUSSION TITLE
404 errors when running sample project

### DISCUSSION THREAD

**Discussion by jovball** [Dotnet-Boxed/Templates#65] [Q&A]
I get nothing but 404 errors when I try to run the sample project. Same result when I create a new project using the template. This happens with both VS 2013 update 4 and VS 2015 update 1.

**Comment by RehanSaeed** [comment]
Presumably this is the MVC 5 template created from the visual studio extension. What URL are you accessing to get 404 errors? Can you give a bit more information about what is happening?

**Comment by jovball** [comment]
From Visual Studio, right-click on any of the views and choose "View in browser" For example, index.cshtml results in the browser opening a URL of /Home/Index. I am running Windows 10 and I get the same result with Chrome, Firefox, Edge, etc.

The error message is generic 404.
The resource cannot be found.
Description: HTTP 404. The resource you are looking for (or one of its dependencies) could have been removed, had its name changed, or is temporarily unavailable.  Please review the following URL and make sure that it is spelled correctly.

Requested URL: /Home/Index

Nothing in the application log tells me what the problem might be. Again, this happens from a new project using the template but I get the same result when I use the sample project in the source code.

**Comment by RehanSaeed** [comment]
Url should be '/' for the home page and not '/Home/Index'. The home and index words are useless, VS tries to be clever and thinks thats most likely where the view will be located. Boilerplate uses cleaner URL's which your users will thank you for, just don't use that VS feature.
